webdriver/tests/bidi/script/add_preload_script/invalid.py

```diff
@@ -186,6 +187,52 @@ async def test_params_arguments_channel_include_shadow_tree_invalid_value(bidi_s
         )
 
 
+@pytest.mark.parametrize("contexts", [False, 42, '_UNKNOWN_', {}])
+async def test_params_contexts_invalid_type(bidi_session, contexts):
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.script.add_preload_script(
+            function_declaration="() => {}",
+            contexts=contexts
+        ),
+
+
+async def test_params_contexts_empty_list(bidi_session):
+    with pytest.raises(error.InvalidArgumentException):
+        await bidi_session.script.add_preload_script(
+            function_declaration="() => {}",
+            contexts=[]
+        ),
+
+
+@pytest.mark.parametrize("value", ["", "somestring"])
+async def test_params_contexts_context_invalid_value(bidi_session, value):
```

Comment:
I actually miss a test for `test_params_contexts_context_invalid_type`. It would be great if you could add that.